docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -205,8 +219,9 @@ All fields in the ``DOCUMENTATION`` block are lower-case. All fields are require
 
 :seealso:
 
-  * A list of references to other modules, documentation or Internet resources
-  * In Ansible 2.10 and later, references to modules must use  the FQCN or ``ansible.builtin`` for modules in ``ansible-core``.
+  * A list of references to other modules, documentation, or internet resources.
+  * Because it stands out better, use ``seealso`` for general references over the use of ``notes`` or adding links to the module ``description``.
```

Comment:
```suggestion
  * Because it is more prominent, use ``seealso`` for general references instead of ``notes`` or adding links to the module ``description``.
```